traffic_portal/app/src/common/modules/form/deliveryService/form.deliveryService.DNS.tpl.html

```diff
@@ -59,7 +59,7 @@
                     <li role="menuitem"><a ng-click="viewOrigins()">Manage Origins</a></li>
                     <li role="menuitem"><a ng-click="viewRegexes()">Manage Regexes</a></li>
                     <li role="menuitem"><a ng-click="viewCapabilities()">Manage Required Server Capabilities</a></li>
-                    <li role="menuitem"><a ng-click="viewServers()">Manage Servers</a></li>
+                    <li ng-if="::deliveryService.topology == null" role="menuitem"><a ng-click="viewServers()">Manage Servers</a></li>
```

Comment:
You should probably do `!deliveryService.topology` instead of using type-coerced null comparison - that way it'll still be disabled if that property is `undefined` or an empty string.

Reply:
yes, i would have but then i could. not use the `::` one way binding syntax.

Reply:
really? That doesn't work?

Reply:
`::! deliveryService.topology` nope

Reply:
actually, looks like this works. testing now. `ng-if="::(!deliveryService.topology)"`

Reply:
nice